include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -1171,7 +1171,7 @@ def AIE_UseLockOp: AIE_Op<"use_lock", []> {
       return 1;
     }
     LockOp getLockOp() {
-      return llvm::cast<xilinx::AIE::LockOp>(getLock().getDefiningOp());
+      return llvm::dyn_cast<xilinx::AIE::LockOp>(getLock().getDefiningOp());
```

Comment:
I think this one is backwards :)

Reply:
I think I didn't mean to change that one actually. Good catch! :)